packages/web/src/components/ui/SearchedResource/SearchedResource.tsx

```diff
@@ -29,8 +37,17 @@ export default function SearchedResource({ image, name, totalPackages, lastScanD
           {name} <span className={styles.searchedResourceHighlight}>{totalPackages} packages</span>
         </h3>
         {!!lastScanDate && (
-          <div className={styles.searchedResourceSubtitle}>
-            Last scanned on {dateTimeFormatter.format(new Date(lastScanDate)).replace(', ', ' at ')}
+          <div>
+            <span className={styles.searchedResourceScanDate}>
+              Last scanned on&nbsp;
+              {dateTimeFormatter.format(new Date(lastScanDate)).replace(', ', ' at ')}
+            </span>
+            {!!onRescanRequested && (
+              <button className={styles.searchedResourceRescanButton} onClick={onRescanRequested}>
+                <Icon kind='refresh' className={styles.searchedResourceRescanButtonIcon} />
+                &nbsp;Update
```

Comment:
probably it would be better to use `white-space: nowrap` instead on `nbsp`s

Reply:
This has nothing to do with `nowrap`, it's an explicit unbreakable `{' '}`